device/config/functions.go

```diff
@@ -0,0 +1,227 @@
+	data := fmt.Sprintf(constants.Language+constants.LocaleLang, storage[GetConstLiteral(Locale)], storage[GetConstLiteral(Locale)])
+
+	_, eut, err := ssh.Run(fmt.Sprintf(`echo "%s" > %s`, data, fp))
+	if err != nil {
+		return errors.New(err.Error() + ":" + eut)
+	}
+
+	fp = help.AddPathSuffix("unix", constants.MountDir, constants.ISAAX_CONF_DIR, "environment")
+	data = fmt.Sprintf(constants.LocaleAll, storage[GetConstLiteral(Locale)])
+
+	_, eut, err = ssh.Run(fmt.Sprintf(`echo "%s" > %s`, data, fp))
+	if err != nil {
+		return errors.New(err.Error() + ":" + eut)
+	}
+
+	return nil
+}
+
+// ConfigLocale is a default method to with dialog to configure the keymap
+func ConfigKeyboard(storage map[string]interface{}) error {
+
+	fmt.Println("[+] Default keyboard: ", constants.DefaultKeymap)
+	if dialogs.YesNoDialog("Change default keyboard?") {
+		l := dialogs.GetSingleAnswer("New keyboard: ", dialogs.EmptyStringValidator)
+		storage[GetConstLiteral(Keymap)] = fmt.Sprintf(constants.KeyMap, l)
+	}
+
+	return nil
+}
+
+// SaveKeyboard is a default method to save KEYMAP into the image
+func SaveKeyboard(storage map[string]interface{}) error {
+
+	if _, ok := storage[GetConstLiteral(Keymap)]; !ok {
+		return nil
+	}
+
+	ssh, ok := storage["ssh"].(ssh_helper.Util)
+	if !ok {
+		return errors.New("Cannot get ssh config")
+	}
+
+	fp := help.AddPathSuffix("unix", constants.MountDir, constants.ISAAX_CONF_DIR, constants.KeyboardF)
+	data := storage[GetConstLiteral(Keymap)]
+
+	_, eut, err := ssh.Run(fmt.Sprintf(`echo "%s" > %s`, data, fp))
+	if err != nil {
+		return errors.New(err.Error() + ":" + eut)
+	}
+
+	return nil
+}
+
+// ConfigWifi is a dialog asking to configure wpa supplicant
+func ConfigWifi(storage map[string]interface{}) error {
+	if dialogs.YesNoDialog("Would you like to configure your WI-Fi?") {
+		storage[GetConstLiteral(Wifi)+"_name"] = dialogs.GetSingleAnswer("WIFI SSID name: ", dialogs.EmptyStringValidator)
+		storage[GetConstLiteral(Wifi)+"_pass"] = []byte(dialogs.WiFiPassword())
+	}
+
+	return nil
+}
+
+// SaveWifi is a default method to save wpa_supplicant for the wifi connection
+func SaveWifi(storage map[string]interface{}) error {
+
+	if _, ok := storage[GetConstLiteral(Wifi)+"_name"]; !ok {
+		return nil
+	}
+
+	ssh, ok := storage["ssh"].(ssh_helper.Util)
+	if !ok {
+		return errors.New("Cannot get ssh config")
+	}
+
+	fp := help.AddPathSuffix("unix", constants.MountDir, constants.ISAAX_CONF_DIR, "wpa_supplicant", constants.WPAsupplicant)
+	data := fmt.Sprintf(constants.WPAconf, storage[GetConstLiteral(Wifi)+"_name"], storage[GetConstLiteral(Wifi)+"_pass"])
+
+	_, eut, err := ssh.Run(fmt.Sprintf(`echo "%s" > %s`, data, fp))
+	if err != nil {
+		return errors.New(err.Error() + ":" + eut)
+	}
+
+	return nil
+}
+
+// ConfigInterface is a dialog asking to setup user Interfaces for the static ip functionality
+func ConfigInterface(storage map[string]interface{}) error {
```

Comment:
__[golint]__
* [golint] func name will be used as config.ConfigInterface by other packages, and that stutters; consider calling this Interface

<!-- Comment by [SideCI](https://sideci.com) -->